analysers/experiments/executionTime.py

```diff
@@ -0,0 +1,71 @@
+import sys
+import json
+import io
+import gzip
+import uuid
+import math
+
+from datetime import timedelta
+
+import scipy.integrate as integrate
+import scipy.special as special
+import numpy as np
+
+from pyspark_cassandra import CassandraSparkContext
+from pyspark_cassandra import RowFormat
+from pyspark import SparkConf
+
+def createQuery(dataRDD, experimentID):
+    from commons import computeMode, computeMetrics
+    
+    queries = []
+    
+    processes = dataRDD.map(lambda a: a["process_definition_id"]).distinct().collect()
+    
+    for process in processes:
+        mode = computeMode(dataRDD.filter(lambda a: a["process_definition_id"] == process).map(lambda r: (r['execution_time'], 1)))
```

Comment:
Why do you filter twice? Just filter one and cache the result if you need it twice